pending/irfanview.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="irfanview">
+  <label>IrfanView</label>
+  <description>Picture Viewer</description>
+  <option id="opendir">
+    <label>Open Directory</label>
+    <description>Directory shown in the open dialog</description>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view32.ini" name="OpenDir"/>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view64.ini" name="OpenDir"/>
+  </option>
+  <option id="mrud">
+    <label>Most recent used directory Thumbnails</label>
+    <description>Most recent used directory (MRUD) - Thumbnails</description>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view32.ini" name="MRUD*"/>
```

Comment:
Wildcard current not supported for `command="ini"`

Reply:
Solved.